### DISCUSSION TITLE
Добавить тексту активной вкладе - bold

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#458] [Есть идея, а что если...]
Прилагаю картинку. Нехватает выделения активной вкладки - можно добавить жирность тексту и линию border-bottom. Что скажете?
![Screenshot_7](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/113771444/cba85fa8-7ff9-480b-9b48-b54204539254)

**Comment by ttm-alexeypolunin** [comment]
нет
